src/main/java/pl/simpleascoding/tutoringplatform/exception/UserIsAlreadyATeacherException.java

```diff
@@ -0,0 +1,11 @@
+package pl.simpleascoding.tutoringplatform.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT)
+public class UserIsAlreadyATeacherException extends IllegalArgumentException {
+    public UserIsAlreadyATeacherException() {
+        super("This is already a teacher");
+    }
+}
```

Comment:
I would recommend including the user's username in the message, as well as extracting the message into a final string field.
Example: `private static final String MESSAGE_USER_ALREADY_A_TEACHER = "User %s is already a teacher";`

Reply:
you're right

Reply:
Agree, cuz we were doing like that in other cases.